app/src/main/java/com/teamagam/gimelgimel/app/view/viewer/cesium/CesiumMapView.java

```diff
@@ -264,6 +270,15 @@ public boolean isReady() {
         return mIsGGMapReadySynchronized.getData();
     }
 
+    @Override
+    public void onError(String error) {
+        if(!mIsHandlingError.getData()) {
+            mIsHandlingError.setData(true);
+            init();
```

Comment:
this event-loop is very heavy and it also run in the uithread which is not acceptable.